src/support.js

```diff
@@ -0,0 +1,220 @@
+"use strict";
+
+const semver = require("semver");
+const currentVersion = require("../package").version;
+
+// Based on:
+// https://github.com/github/linguist/blob/master/lib/linguist/languages.yml
+
+const supportTable = [
```

Comment:
Maybe we should add a script to generate this table from `linguist` so that it'd be easy to update in the future.

Reply:
Maybe. I ended up changing quite a few things though, added more extensions, filenames, added groups, etc.